frontend/src/layout/header/UserBox.tsx

```diff
@@ -82,7 +82,7 @@ const LoggedOut: React.FC = () => {
                         color: "var(--nav-color-bw-contrast)",
                     },
                     ":hover": { outline: "2px solid var(--grey80)" },
-                    ":focus": { outline: "2px solid var(--accent-color)" },
+                    ...focusStyle({ offset: 1 }),
```

Comment:
The `outlineOffset` a few lines up can be removed then, right?

Reply:
Yes. This was still used for the hover outline, but I'm debating whether the login button should even have that outline, as it appears to be the only text button with it. I believe I initially added it since the neighboring icon buttons and the search field also have that outline on hover.

So I think I should either remove the hover outline from this or add it to the other text buttons. But in any case, it doesn't need to have an offset.

Reply:
I think removing the hover outline is fine